### ISSUE TITLE
v 6.0 couple bugs

### ISSUE THREAD

**Issue by brjuseg** [galaipa/SuperBuildBattle#16]
Starting new topic, cause it`s already 6.0 :)
Tried it just now, still having same issue with conflicts with other plugins:

[04:48:44] [Server thread/WARN]: [WorldBorder] Border-checking task was not running! Something on your server apparently killed it. It will now be restarted.
[04:48:44] [Server thread/INFO]: [WorldBorder] [CONFIG] Border-checking timed task stopped.
[04:48:44] [Server thread/INFO]: [WorldBorder] [CONFIG] Border-checking timed task started.

It says that task started, but not all conflicting plugins can restart their tasks... another one (infernal mobs) can`t.
And armor disappeared again.
I am using 1.8.7 spigot and 6.0 SBB.

Thx and welcome to my server to test it if you want :)

**Comment by galaipa** [comment]
I hope is fixed now. Can you confirm it please?

**Comment by brjuseg** [comment]
Armor problem fixed in 6.5. Other plugins tasks still conflict.

**Comment by galaipa** [comment]
I promise you I will work tomorrow on it and send you a beta

**Comment by galaipa** [comment]
I think I have fixed it. Try it and let me know please :D
http://www.mediafire.com/download/azpp6gf6boyq9w2/SuperBuildBattle.jar

**Comment by brjuseg** [comment]
I`ve checked and strange things happen now :)

 

1. Game not starting automatically (i usually try it with 2 players, now i had to press forcestart to start the game).

2. Equipped armor disappear from the player the moment he types in "bb join". And not returning after the game. But if you press "bb join" and then "bb leave" before game starts, you can have your armor back.

 

Actually, version 6.5 works well enough if one use it on dedicated server in bungee mode. Cause you don`t have to worry about world border, armor and other survival stuff. Unfortunately, not all (and not me) actually have bungee servers :P (But i plan to restructure my server for bungee sometime later).

 

One thing your great plugin misses - multiarena support. Or i failed to do multiple arenas xD

 

Best wishes, Gleb.

**Comment by galaipa** [comment]
So the task conflicts are fixed? I have fixed the problem with the minimun players ( I accidentaly removed a line from the code) But I don't know what happens with the armor, it should work...

http://www.mediafire.com/download/azpp6gf6boyq9w2/SuperBuildBattle.jar

**Comment by brjuseg** [comment]
Yes, tasks conflict is fixed. I`ll check again armor thingy somewhere around next morning :) What i remember from this morning is: if you use /bb leave, you get your armor back. If the game ends by itself - you don`t.

 

Cya around, best wishes.
